src/2_RTB/VerbaliInterni/VerbaleInterno_211123/content.typ

```diff
@@ -0,0 +1,41 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+In data odierna sono stati trattati i seguenti temi.
+
+== Frequenza degli aggiornamenti alla Proponente
+Il team si impegna a mantenere la Proponente costantemente informata sull'avanzamento durante lo Sprint, con un aggiornamento settimanale. Tale aggiornamento avverrà idealmente a metà Sprint; tuttavia, in caso di urgenze o necessità particolari, il team contatterà la Proponente per ottenere il loro _feedback_ in momenti aggiuntivi. Inoltre, comprendendo l'interesse dei referenti della Proponente nell'analizzare il codice sviluppato prima della _Sprint Retrospective_, il team si impegna a contattarli anche poco prima della fine dello Sprint, immediatamente dopo aver completato il _merge_ del _feature branch_ di interesse nel _branch_ di derivazione nella _repository_ InnovaCity.
```

Comment:
L'ultima frase e' un po' troppo specifica: se in futuro cambiamo metodologie a livello di NdP questa frase risulta non vera; toglierei dall'ultima virgola in poi.